src/App/App.tsx

```diff
@@ -12,6 +12,7 @@ import { IconButton } from '../components/IconButton';
 import { HelpIcon } from '../components/icons/HelpIcon';
 
 const Layout = styled.main`
+  isolation: isolate;
```

Comment:
Because of the isolation,  any components in `<Layout />`are never overlain components outside it's stack context.